graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountTweetRecsGenerator.java

```diff
@@ -130,22 +52,22 @@ private static boolean isLessThantMinUserSocialProofSize(
     int minUserSocialProofSize =
```

Comment:
It might make sense to refactor this into a function in GeneratorUtils, since it is similarly used in the User Generator